src/components/Input/Input.js

```diff
@@ -4,7 +4,9 @@ import {COLORS} from '../../utils/theme';
 
 const {width} = Dimensions.get('screen');
 
-const Input = ({...props}) => <TextInput style={styles.input} {...props} />;
+const Input = ({...props}) => (
+  <TextInput style={styles.input} keyboardAppearance="dark" {...props} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made here is adding the `keyboardAppearance="dark"` prop to the TextInput component. This will make the keyboard have a dark appearance when this input field is focused. However, it's important to note that this prop only works on iOS. If your application also targets Android, you should consider making this prop conditional based on the platform.
```diff
- <TextInput style={styles.input} keyboardAppearance="dark" {...props} />
+ <TextInput style={styles.input} keyboardAppearance={Platform.OS === 'ios' ? "dark" : undefined} {...props} />
```
This way, the `keyboardAppearance` prop will only be applied if the platform is iOS, and will be ignored on Android where it's not supported.


<!-- This is an auto-generated comment by OSS CodeRabbit -->